src/app/donation-start/donation-start.component.ts

```diff
@@ -715,10 +715,10 @@ export class DonationStartComponent implements AfterContentChecked, OnDestroy, O
     });
   }
 
-  private redirectToEnthuse(donation: Donation) {
+  private redirectToEnthuse(donation: Donation, logoUri: any) {
```

Comment:
Type can be `?string`?

Reply:
Hmm doesn't seem to like it @NoelLH, getting this `Argument of type 'string | undefined' is not assignable to parameter of type 'string'. Type 'undefined' is not assignable to type 'string'.`

Reply:
@tbg-jasonfung it should be fine as long as the optional bit (`?` / `undefined`) is consistent across methods

Reply:
Ahh cool, got it!